### PR TITLE
Uploading drafts for volunteer tutorial and board game night

### PR DESCRIPTION
**Pull request by jcjudkins**
## Description

Blog posts

Added post for Volunteering
Added draft for Game Night
Added draft for Tutorials
Added images

### CODE REVIEW

File: _drafts/2023-09-12-board-game-night.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+We're so excited to tell you about your opportunities to socialize with and get to know other attendees, speakers, organizers, and sponsors!
+
+
+## Board Game Night
+
+**When**: 8:00-10:00 pm, Tuesday, October 17
+
+**Where**: TBD
```
Comment (drewbrew) [inline _drafts/2023-09-12-board-game-night.md:17]: This will almost certainly be in the junior ballroom

**Review by drewbrew — APPROVED** [review]

**Comment by tim-schilling** [comment]
How's this @jcjudkins 
image

**Review by drewbrew — APPROVED** [review]

**Comment by tim-schilling** [comment]
Okay [0c567ec](https://github.com/djangocon/2023.djangocon.us/pull/191/commits/0c567ec024401ec6dde8ad48794d0ac305555699) gets back to images on an entire line. It adds some alt text and uses classes for the styles. It also uses `<p>` elements rather than `<div><br>` for more consistent spacing to match the other lines. I thought the wrapped text looked nice, but I'm happy with either.

**Comment by jcjudkins** [comment]
The way you changed it actually does look nice. When I initially did that the text was pushing up against the image and I found it very annoying. However, what you showed the text is spaced much better. I think I like it that way as well! 

Thank you!

**Comment by jcjudkins** [comment]
Updated Picture and publish date.

**Pull request merged**